src/pages/index.html

```diff
@@ -47,38 +61,48 @@
                 </article>
 
                 <article class="homepage-ride-info-article-container">
+                    <h2>Food and Drink</h2>
                     <img src="../public/images/img1.jpg" class="homepage-ride-info-section-img">
-                    <button class="homepage-ride-info-section-button">Link button</button>
-                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+                    <span>Terry the Trapezoid’s tea party is fun for all the family! Check out our shape-themed menus and keep the fun going through lunch!</span>
+                    <button class="homepage-ride-info-section-button">More</button>                
                 </article>
             </section>
 
             <!-- Theme park description -->
             <section class ="homepage-park-description-container">
                 <div class="homepage-park-description-text-container">
-                    <p><b>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam venenatis fermentum euismod. Praesent in nunc ut augue egestas blandit nec eget orci. Vestibulum quis ullamcorper nulla, sit amet imperdiet augue. Curabitur ac massa felis. Integer et arcu in sapien accumsan consectetur nec lobortis leo. Maecenas tincidunt laoreet justo ac accumsan. In sit amet rutrum turpis.</b></p>
-                    <button class="homepage-park-description-button">Link button</button>
+                    <h3>Our Ethos</h3>
+                    <p>Shapeland is the world's first theme park dedicated solely to shape-recognition in the early years. Experts agree that shape-recognition 
+                        is an important pre-cursor to later literacy and numeracy skills. Enter our magical world and your little ones will earn all about geometric 
+                        shapes through fun and engaging rides and activities! Even our food and drink is shape-themed! But most importantly, our family-focused 
+                        facilities are world-renowned with baby pit stop stations in accessible locations for feeding and changing, quiet areas for naps, drinking 
+                        water fountains across the park, and a range of healthy snack stations at every turn. Learn more about how our facilities support families 
+                        with young children here:
+                    </p>
+                    <button class="homepage-park-description-button">Our Family-Centered Facilities</button>
                 </div>
                 <img src="../public/images/img3.jpg" width="550">
             </section>
 
             <!-- Link buttons -->
             <section class ="homepage-shape-links-container">
-                <img src="../public/images/circle.png">
-                <img src="../public/images/triangle.png">
-                <img src="../public/images/square.png">
+                <img src="../public/images/circle.png" class="homepage-shape-links-container-img">

```

Comment:
`homepage-shape-links-container-img`  is not a container, it doesn't have any elements inside of the `img`  tag that should behave as flex items and it's not needed for the `:hover`  effect as well.
In this case it's much better to apply directly to `img` tags inside the `homepage-shape-links-container` class:
```
.homepage-shape-links-container > img:hover {
	animation: shape-img-ani-1 2s ease 0s 1 normal forwards;
}
```